prometheus.yml/prometheus.yml

```diff
@@ -0,0 +1,14 @@
+global:
+  scrape_interval: 30s
+  evaluation_interval: 25s
+
+scrape_configs:
+  - job_name: "backend"
+    static_configs:
+      - targets: ["localhost:8081"]
+
+remote_write:
+  - url: https://prometheus-prod-24-prod-eu-west-2.grafana.net/api/prom/push
+    basic_auth:
+      username: 1433805
+      password: glc_eyJvIjoiMTA1NzgzNCIsIm4iOiJpaHAtaWhwIiwiayI6ImVsa2Q2bGRYOTAyMzY1MlcxNmNJU3ZEUSIsIm0iOnsiciI6InByb2QtZXUtd2VzdC0yIn19
```

Comment:
Make sure to not include credentials in commits. Revoke the credentials or change the password.